readme.txt

```diff
@@ -188,6 +188,10 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= [5.4.2] TBD =
+
+* Fix - No need to promoter ET+ if it's already installed. [ET-1528]
```

Comment:
`promoter`. Maybe we can try to be more specific with the readme entry? For example, explaining where the change is: "Removed Event Tickets Plus upsell text on Tickets Commerce when Event Tickets Plus is installed and active".

Reply:
We should also try to avoid using acronyms in readme entries. 

ET+ => Event Tickets Plus
ARF => Attendee Registration Fields

And so on.